best-practices/client-side/framework/react/README.md

```diff
@@ -1,3 +1,25 @@
-# react
+# React
 
-## all info on current dev with react, how to spin up a boilerplate
\ No newline at end of file
+## Intro
+
+-   [React.js](https://reactjs.org/docs/getting-started.html) often referred to as React or ReactJS is an open-source JavaScript library for building user interfaces or UI components, created by Facebook.
+
+-   [React Native](https://reactnative.dev/) is an open-source mobile application framework created by Facebook to develop applications for Android, iOS, Web and UWP by enabling developers to use React along with native platform capabilities.
+
+## Examples of react based projects
+
+To get more familiar with the structure of a React/React Native based project, below is a list of some repositiories you can look into. If you currently don't have access to any of these projects, please reach out and request access:
+
+-   [bw-project-status](https://github.com/Shift3/bw-project-status)
+-   [rn-by-example](https://github.com/Shift3/rn-by-example)
+-   [Aerostar](https://github.com/Shift3/Aerostar)
+-   [sundial-mobile-app](https://github.com/Shift3/sundial-mobile-app)
```

Comment:
You may want to specify which of these are made with reactjs and react-native. Actually, it might be better to have this page split out between the two, since they both have different ways of setup and development. It might be a bit out of scope, but could be a great followup item.